src/test/java/org/jabref/TestIconsProperties.java

```diff
@@ -31,15 +31,14 @@ public void testExistenceOfIconImagesReferencedFromIconsProperties() throws IOEx
         try (Reader reader = Files.newBufferedReader(Paths.get(iconsPropertiesPath))) {
             properties.load(reader);
         }
-        assertFalse("There must be loaded properties after loading " + iconsPropertiesPath,
-                properties.entrySet().isEmpty());
+        assertFalse(properties.entrySet().isEmpty(), () -> "There must be loaded properties after loading " + iconsPropertiesPath);
```

Comment:
Hm, not sure if I like these Supplier Lambdas...Are they really needed? Makes the methods look more complex and I doubt that the evaluation of the message really takes a long time. WDYT?

Reply:
In the beginning I did not see that I can pass the message as parameter without a supplier . The later tests are without. The supplier is only needed in the assertThroes